draft-dukhovni-tls-dnssec-chain.xml

```diff
@@ -901,6 +902,27 @@ _dane443.node1.provider.example. IN TLSA 1 1 1 ...
       extension are described in <xref target="pinning"/>.
     </t>
 
+    <t>
+      Each zone in the DANE chain validation path for a domainname can
+      impersonate the server to any client.  A mitigation is for clients
+      to prefer or require TLSA usage 2, binding a WebPKI issuer as the
```

Comment:
s/prefer or//, and that would be usages `PKIX-TA(0)` and `PKIX-EE(1)`.

Reply:
The impersonation happens in DNS, not in the TLSA chain only. If the server generates the chain from public data, it is the public DNS data that is "wrong", but one might of course argue that the parents are always "right". If you have a rogue parent, no TLSA usage won't help you and adding trust into webpki parties just dilutes the trust and increases the attack surface by all the CAs - something we are explicitly trying to move away from with a DNSSEC based PKI. So I am opposed to introducing that now here.